rpc/wrtc_signaling_answerer.go

```diff
@@ -478,7 +478,12 @@ func (ans *webrtcSignalingAnswerer) answer(client webrtcpb.SignalingService_Answ
 		return err
 	}
 	if err := sendDone(); err != nil {
-		return err
+		// Errors from sendDone (such as EOF) are sometimes caused by the signaling
+		// server "ending" the exchange process earlier than the answerer due to
+		// the caller being able to establish a connection without all the
+		// answerer's ICE candidates (trickle ICE). Only Warn the error here to
+		// avoid accidentally Closing a healthy, established peer connection.
```

Comment:
[minor] unless you purposely capitalized to refer to functions

```suggestion
		// answerer's ICE candidates (trickle ICE). Only warn about the error here to
		// avoid accidentally closing a healthy, established peer connection.
```

Reply:
Ah I did mean to refer to `logger.Warn` and `peerConnection.Close`, so might leave the caps.